fuseops/ops.go

```diff
@@ -500,6 +500,14 @@ type OpenDirOp struct {
 	// a later call to ReleaseDirHandle.
 	Handle    HandleID
 	OpContext OpContext
+
+	// CacheDirContentAsPageCache conveys the kernel to cache the response of next
+	// ReadDirOp as page cache. Once cached, listing on that directory will be
+	// served from the kernel until invalidated.
+	CacheDirContentAsPageCache bool
```

Comment:
Please rename this field to `CacheDir` for consistency with the Linux kernel flag name (`FOPEN_CACHE_DIR`). That the page cache is used for caching is an implementation detail.